modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/thin_client_test/ThinClientCachePut.java

```diff
@@ -48,10 +48,10 @@ protected void run(JsonNode jsonNode) throws Exception {
         ClientCacheConfiguration cfg = new ClientCacheConfiguration();
         cfg.setName(cacheName);
 
-        System.out.println(">>> Cache requested [cacheName=" + cacheName + ']');
-        System.out.println(">>>                 [cacheMode=" + cacheMode + ']');
-        System.out.println(">>>                 [cacheAtomcityMode=" + cacheAtomcityMode + ']');
-        System.out.println(">>>                 [backups=" + backups + ']');
+        System.out.println(">>> Cache requested  [cacheName=" + cacheName + ']');
```

Comment:
We should never use System.out.println, use a logger instead.
We should also avoid logging using some messages for a single event, this approach does not work in the case of concurrency.